docs/source/getting-started/content/con_understanding-codeready-containers.adoc

```diff
@@ -2,6 +2,8 @@
 = Understanding {prod}
 
 {rh-prod} brings a minimal OpenShift 4.0 or newer cluster to your local computer.
+This cluster provides a minimal environment for development and testing purposes.
+It may or may not be a good fit for other usecases, and is not a replacement for the https://cloud.redhat.com/openshift/install/[full-fledged OpenShift installer].
```

Comment:
this statement is right, but sounds harsh... I am not sure how to rephrase; tried to reverse but that doesn't sound good about what this is about, but I am kinda inclined to mention a "headless, multi-developer setup" / "team-based"

Reply:
Ah forgot to change that part, I'll update the PR a second time

Reply:
I currently have:
> Red Hat CodeReady Containers brings a minimal OpenShift 4.0 or newer cluster to your local computer. This cluster provides a minimal environment for development and testing purposes. It’s mainly targetted at running on developers' desktops. For other use cases, such as headless, multi-developer/team-based setups, …​, use of the full-fledged OpenShift installer is recommended.